modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -1829,9 +1830,8 @@ public void request(long n) {
                 if (n <= 0) {
                     cancel(null, true);
 
-                    subscriber.onError(new IllegalArgumentException(IgniteStringFormatter
-                            .format("Invalid requested amount of items [requested={}, minValue=1]", n))
-                    );
+                    subscriber.onError(new IllegalArgumentException(
+                            format("Invalid requested amount of items [requested={}, minValue=1]", n)));
```

Comment:
```suggestion
                            format("Invalid requested amount of items [requested={}, minValue=1].", n)));
```

Reply:
We don't set a final dot.

Reply:
it's not the true - https://cwiki.apache.org/confluence/pages/diffpagesbyversion.action?pageId=199534707&selectedPageVersions=15&selectedPageVersions=14